client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -180,7 +180,13 @@ class SimplePaymentsDialog extends Component {
 		return new Promise( resolve => this.formStateController.handleSubmit( resolve ) );
 	}
 
-	handleChangeTabs = activeTab => this.setState( { activeTab } );
+	handleChangeTabs = ( activeTab, why ) => {
+		if ( why === 'add' ) {
```

Comment:
Is the `why` parameter really necessary? The `activeTab` is either `list`, and in that case we want to always show the list. Or it is `form`, in which case we want to display the "Add New" form. No other cases are there.